docs/topics/coroutines.rst

```diff
@@ -85,13 +87,16 @@ This means you can use many useful Python libraries providing such code::
         async def parse_with_asyncio(self, response):
             async with aiohttp.ClientSession() as session:
                 async with session.get('https://additional.url') as additional_response:
-                    additional_data = await r.text()
+                    additional_data = await additional_response.text()
             # ... use response and additional_data to yield items and requests
 
 .. note:: Many libraries that use coroutines, such as `aio-libs`_, require the
           :mod:`asyncio` loop and to use them you need to
           :doc:`enable asyncio support in Scrapy<asyncio>`.
 
+.. note:: If you want to ``await`` on Deferreds, you may need to
+          :ref:`wrap them<asyncio-await-dfd>`.
```

Comment:
Does it mean that the example above is incorrect? If asyncio is enabled, parse_with_deferred won't work; if asyncio is not enbaled, parse_with_asyncio won't work?

Reply:
@kmike we can split the example into two spiders or we can maybe replace aiohttp with something that doesn't require the asyncio reactor, what do you think?

Reply:
+1 to split, feature both scenarios.